ast/misc.go

```diff
@@ -318,10 +318,12 @@ func (n *FlushStmt) Accept(v Visitor) (Node, bool) {
 type KillStmt struct {
 	stmtNode
 
+	// Query ...
```

Comment:
Query indicates whether terminate a single query on this connection or the whole connection.